src/uu/chmod/src/chmod.rs

```diff
@@ -206,13 +207,23 @@ impl Chmoder {
                         ));
                     }
                 } else if !self.quiet {
-                    return Err(USimpleError::new(
-                        1,
-                        format!(
-                            "cannot access {}: No such file or directory",
+                    if files.len() == 1 {
+                        return Err(USimpleError::new(
+                            1,
+                            format!(
+                                "cannot access {}: No such file or directory",
+                                filename.quote()
+                            ),
+                        ));
+                    } else {
+                        println!(
+                            "{}: cannot access {}: No such file or directory",
+                            uucore::util_name(),
                             filename.quote()
-                        ),
-                    ));
+                        );
+                        set_exit_code(1);
+                        continue;
+                    }
                 }
                 return Err(ExitCode::new(1));
```

Comment:
Have you checked the behaviour for these other `return`s? They might also not exit in GNU.